Carousel: use buttons, not links, for prev/next controls (#32627)
twbs/bootstrap: site/content/docs/5.0/components/carousel.md

### With controls

> Adding in the previous and next controls. We recommend using `<button>` elements, but you can also use `<a>` link elements–though in that case, you should also add `role="button"` attributes to the links.

Reviewer: Can make this short and sweet IMO.

```suggestion
Adding in the previous and next controls. We recommend using `<button>` elements, but you can also use `<a>` elements with `role="button"`.
```

Author: that might be misleading, as it may suggest our code only supports `<a>` with `role="button"`, while in effect it doesn't really check for `role="button"`?

Author: @mdo would you be ok with the more verbose, but possibly clearer, wording?

Reviewer: I think brevity and best practice should win out—if folks use anchor elements, they really should use `role="button"`, yes? If that's not the case, then yeah we can use the more verbose.

Author: *shrug* fairy snuff :)
